### PR TITLE
SQL

### PR DESCRIPTION
**Pull request by kourtbailey**
Hello @benjamenalford ; some fresh SQL is ready to merge!

### CODE REVIEW

File: something.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SELECT username
+FROM users
+WHERE username ILIKE '%foo%';
```
Comment (benjamenalford) [inline something.sql:3]: @kourtbailey ILIKE? does that work?

Comment (kourtbailey) [reply something.sql:3]: Yes, https://www.postgresql.org/docs/current/functions-matching.html

Comment (benjamenalford) [reply something.sql:3]: New to me, thanks!

**Review by benjamenalford — APPROVED** [review]
Thanks for showing me that!  looks good

**Pull request merged**